docs/docker.md

```diff
@@ -0,0 +1,40 @@
+# Quickstart with Docker
+
+If you want to try running .NET Spark applications without worrying about what to download and install, you can try using Docker instead.
+
+## Build
+To build the image, you can run:
+
+```bash
+DOCKER_BUILDKIT=1 docker build -t dotnet-spark .
+```
+
+This will create a `dotnet-spark` image.
+
+## Running a Script
+If you have already built your project and all that's left is to run it in an environment with a .NET Spark setup, then you can run:
+
```

Comment:
I think it would be more clear if we have two different sections (visually separate them) - one for directly running it vs. one for build/run.

Reply:
Is that okay? They'll have almost the same Dockerfile content. Maybe we can create scripts instead?